src/helpers/cells/cell_types.ts

```diff
@@ -282,7 +282,7 @@ export class FormulaCell extends AbstractCell implements IFormulaCell {
       switch (typeof value) {
         case "number":
           return {
-            value,
+            value: value || 0, // necessary to avoid "-0" values
```

Comment:
ok for me

Reply:
my 2 cents; in this case;  the limitation lies in the test framework. Unless we want to patch toBe, toEqual , etc... I think it' reasonable to go with this

Reply:
I'd also argue that it's possible for us in the future to use `Object.is()` to test an equality somewhere in the business code, which won't always work because of the -0. I don't see a situation where we would need to differenciate between 0 and -0.

Reply:
![js](https://user-images.githubusercontent.com/10908305/194571906-8abcc265-8a4b-4ee5-a526-b121be0c8d5d.gif)